Tests/AMPUtilsTests.m

```diff
@@ -0,0 +1,45 @@
+//
+//  AMPUtilsTests.m
+//  Amplitude
+//
+//  Created by Qingzhuo Zhen on 2/5/24.
+//  Copyright © 2024 Amplitude. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "AMPUtils.h"
+
+@interface AMPUtilTests : XCTestCase
+
+@end
+
+@interface FakeAMPUtils: AMPUtils
+@end
+
+@implementation FakeAMPUtils
+
++ (NSDictionary *)getEnvironment {
+    return @{@"APP_SANDBOX_CONTAINER_ID": @"test-container-id"};
+}
+
+@end
+
+@implementation AMPUtilTests {}
+
+- (void) testIsSandboxEnabled {
```

Comment:
I think it would be good to [add some macos specific tests similar to Swift](https://github.com/amplitude/Amplitude-Swift/pull/113/files#diff-6bc0924377efa496272cf54a320ad4c3ecb5e4b273760c3f3fb85609f20bcbf0R52-R75). But not sure how hard it is to add macos specific tests to this repo.

Reply:
It is already tested in Swift, so not 100% necessary.